chat-plugins/poll.js

```diff
@@ -406,6 +414,31 @@ exports.commands = {
 		`/poll display [poll id number] - Displays the poll. The poll id number is optional for this command and displays only the poll with the matching id number.`,
 		`/poll end [poll id number] - Ends a poll and displays the results. The poll id number is optional for this command and ends only the poll with the matching id number. and Requires: % @ * # & ~`,
 	],
+
+	tpoll: 'tierpoll',
+	tierpoll: function (target, room, user, connection, cmd, message) {
+		if (!this.can('minigame', null, room)) return false;
+		if (room.poll && room.poll.pollArray.length >= 5) return this.errorReply("The maximum amount of polls is 5.");
+		let options = [];
+		for (let key in Dex.formats) {
+			if (!Dex.formats[key].mod) continue;
+			if (!Dex.formats[key].searchShow) continue;
+			if (toId(target) !== 'all') {
+				let commonMods = ['gen7', 'wlssb', 'pmd', 'cssb', 'metronome', 'digimon'];
+				if (commonMods.indexOf(Dex.formats[key].mod) === -1) continue;
+			}
+			options.push(Dex.formats[key].name);
+		}
+		room.poll = new Poll(room, {
+			source: 'What should the next tournament tier be?',
+			supportHTML: false,
+			username: user.name,
+		}, options);
```

Comment:
Needs to be setup to work with multi-polls

Reply:
It does?

Reply:
... yes because otherwise your removing all other polls and making a new one will crash the server.
We don't have normal polls anymore...

Reply:
My bad with my testing I never saw that